modules/framework/galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -2742,6 +2742,7 @@ components:
             anyOf:
               - $ref: '#/components/schemas/GalasaProperty'
               - $ref: '#/components/schemas/GalasaSecret'
+              - $ref: '#/components/schemas/Stream'
```

Comment:
Minor - could this be GalasaStream to be more consistent with the existing resource definitions in the openapi docs?